nifi-nar-bundles/nifi-framework-bundle/nifi-stateless-bundle/nifi-stateless-api/src/main/java/org/apache/nifi/stateless/flow/StatelessDataflow.java

```diff
@@ -24,8 +24,33 @@
 import java.util.Set;
 
 public interface StatelessDataflow {
```

Comment:
It is not part of this PR, but wouldn't make sense to make it extend AutoClosable? and having close() instead of shutdown(), etc.